dcs-packaging-tool/dcs-packaging-tool-model/src/main/java/org/dataconservancy/packaging/tool/model/dprofile/PropertyValue.java

```diff
@@ -0,0 +1,30 @@
+package org.dataconservancy.packaging.tool.model.dprofile;
+
+import java.util.List;
+
+/**
+ * The value of a property.
+ */
+public interface PropertyValue {
+    /**
+     * @return The type of the property.
+     */
+    PropertyType getPropertyType();
+
+    /**
+     * @return If type is string, return string value.
+     */
+    String asString();
```

Comment:
Difference from `toString()`?  If the property can't be represented by a string then what is the behavior of this method?